packages/flutter/lib/src/widgets/draggable_scrollable_sheet.dart

```diff
@@ -317,13 +317,30 @@ class _DraggableScrollableSheetState extends State<DraggableScrollableSheet> {
       }
       _extent._currentExtent.value = _extent.initialExtent;
     }
+    _child = widget.builder(context, _scrollController);
+  }
+
+  @override
+  void didUpdateWidget(DraggableScrollableSheet oldWidget) {
+    super.didUpdateWidget(oldWidget);
+    _child = widget.builder(context, _scrollController);
+    _updateExtent();
```

Comment:
I think `_updateExtent` has to run before we execute the builder since it instantiates a new scrollController that's used by the builder?

Reply:
Done.